AI/DB_interface.py

```
@@ -0,0 +1,54 @@
+        print(f"Error in get_data: {e}") # prints the error
+        close(conn) # closes the connection
+        return [] # returns an empty list
+    
+def execute_query(query, params=()):
+    conn = connect() # connects to the DB
+    try:
+        cursor = conn.cursor() # creates a cursor object
+        cursor.execute(query, params) # executes the query with params
+        conn.commit() # commits the changes
+        close(conn) # closes the connection
+        return True
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)